tox.ini

```diff
@@ -1,16 +1,16 @@
 [tox]
-envlist = flake8, pydocstyle, py27, py35
+envlist = py36
```

Comment:
if you use the magical environment `py` then it will default to whatever version of python is available.

Reply:
Thank you for your input. My question then is - should we do that, or make sure it works well with a specific version(s) so that we can advise that to whoever wants to install?

Reply:
Both are possible - Travis can test on several versions of python, and it looks like you've already put them in the `.travis.yml`. Further, you can see from the last build log at https://travis-ci.org/nunogit/scholia/jobs/633355631 that there's a mismatch between the version of python available when specifying it this way, and the version tox wants to run.

However, I would suggest removing python34. It's really old and most libraries aren't supporting it anymore (see https://travis-ci.org/nunogit/scholia/jobs/633355625)

Reply:
Ok, now I understand what you mean. Agreed!